src/lib/KStorage.ts

```diff
@@ -76,6 +78,11 @@ export class KStorage {
 	  return Promise.resolve(true);
 	}
 
+  start(subscription: Subscription<any>) {
```

Comment:
Missing return types and can we include doc block comments too please.

Reply:
Added these